scripts/lint-migration-journal.mjs

```
@@ -0,0 +1,103 @@
+function hashOf(tag) {
+  return createHash("sha256")
+    .update(readFileSync(join(migrationsDir, `${tag}.sql`), "utf8"))
+    .digest("hex");
+}
+
+// `--hashes` prints the tag/stamp/hash table an operator needs to read a
+// `drizzle.__drizzle_migrations` row back to the migration it stands for.
+// Repairing a stranded install means editing one bookkeeping row, and the
+// row only carries a hash — without this mapping, identifying the right
+// one is guesswork. See docs/DEPLOYMENT.md.
+if (process.argv.includes("--hashes")) {
```

ESLint

[error] 'process' is not defined.

(no-undef)